tests/phpunit/tests/blocks/getBlockTemplates.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * @group blocks
```

Comment:
As the existing [tests/block-template-utils.php](https://github.com/WordPress/wordpress-develop/blob/eafd5201cb2aa295531830745e0960a12c415af8/tests/phpunit/tests/block-template-utils.php) is part of `@group block-templates`, should this test be part of that group?

I'm not sure where/how the "rules" regarding groups are determined, so I honestly don't know.

🙌🏻 Kudos for putting this into a separate test file!

Reply:
Thanks for the review, @ironprogrammer.
Good catch!
Fixed in 66946f6df5fe9b950503aa38df30c416f6ed2f30